nsiqcppstyle_rulemanager.py

```diff
@@ -248,8 +248,9 @@ def __init__(self):
         __builtins__["__import__"] = self._import
         self.newModules = {}
 
-    def _import(self, name, globals=None, locals=None, fromlist=[]):
-        result = self.realImport(*(name, globals, locals, fromlist))
+    def _import(self, *args, **kwargs):
+        name = args[0]

```

Comment:
No guarantee that `args[0]` exists. Please use name as an explicit parameter

Reply:
Good catch.

Done.